src/main/webapp/resources/js/apis/ontology/taxonomy/query.ts

```diff
@@ -2,7 +2,13 @@ import axios from "axios";
 import { setBaseUrl } from "../../../utilities/url-utilities";
 
 const TAXONOMY = "taxonomy";
-const ONTOLOGY_URLS = { [TAXONOMY]: setBaseUrl("ajax/taxonomy") };
+const ONTOLOGY_URLS: ontologyDictionary = {
+  [TAXONOMY]: setBaseUrl("ajax/taxonomy"),
+};
+
+export interface ontologyDictionary {
```

Comment:
Rename to `OntologyDisctionary` - interfaces should be capitalized

Reply:
Fixed in 1398b80e9ff839bad02c6c0dfd63fabd2e3bf8a5